crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -323,8 +322,13 @@ where
     /// If automatically inserting [`apply_deferred`](crate::schedule::apply_deferred) like
     /// this isn't desired, use [`after_ignore_deferred`](Self::after_ignore_deferred) instead.
     ///
-    /// Note: The given set is not implicitly added to the schedule when this system set is added.
-    /// It is safe, but no dependencies will be created.
+    /// # Note
+    ///
+    /// If you configure two groups of systems (e.g. 'Set A') in a schedule, and another group references `.after` or `.before` (e.g. 'Set B'), the systems in Set B will not be automatically scheduled.
+    ///
+    /// This means that Set A and Set B will run independently unless they are explicitly linked. So `.after`/`.before` will not provide the desired behaviour, so the Sets can run in parallel or in any order determined by the scheduler.
```

Comment:
```suggestion
    /// This means that Set A and Set B will run independently unless they are explicitly linked. So `.after`/`.before` will not provide the desired behaviour, so the Sets can run in parallel or in any order determined by the scheduler.
   /// Only use `after(B)` and `before(B)` when you know that `B` has already been scheduled for you, e.g. when it
   /// was provided by Bevy or a third-party dependency, or you manually scheduled it somewhere else in your app.
```

Reply:
That's great. This was missing information. As mentioned earlier the (example shown in docs)[https://docs.rs/bevy/latest/bevy/ecs/prelude/trait.IntoSystemConfigs.html#method.before] did not help me a lot.